cmd/layotto/main.go

```diff
@@ -269,14 +277,17 @@ func NewRuntimeGrpcServer(data json.RawMessage, opts ...grpc.ServerOption) (mgrp
 
 		// File
 		runtime.WithFileFactory(
-			file.NewFileFactory("aliOSS", alicloud.NewAliCloudOSS),
+			file.NewFileFactory("aliyunOSS", aliyun.NewAliyunFile),
 			file.NewFileFactory("minioOSS", minio.NewMinioOss),
-			file.NewFileFactory("awsOSS", aws.NewAwsOss),
+			file.NewFileFactory("awsOSS", aws.NewAwsFile),
 			file.NewFileFactory("tencentCloudOSS", tencentcloud.NewTencentCloudOSS),
 			file.NewFileFactory("local", local.NewLocalStore),
 			file.NewFileFactory("qiniuOSS", qiniu.NewQiniuOSS),
 		),
-
+		runtime.WithOssFactory(
+			oss.NewOssFactory("aws.oss", aws2.NewAwsOss),
```

Comment:
`aws2` is not a good name. How about `aws_oss`?

Reply:
done